example/javascriptlib/basic/5-client-server-hello/build.mill

```diff
@@ -0,0 +1,62 @@
+package build
+import mill._, javascriptlib._
+
+object client extends JestModule
+
+object server extends JestModule
+
+object app extends TypeScriptModule {
+  def moduleDeps = Seq(client, server)
+
+  def bundleApp =
+    Task {
+      val serverPath = server.bundle().path
+      val clientPath = client.bundle().path
+      println(s"client bundled@$clientPath\nserver bundled@$serverPath")
+      (PathRef(serverPath), PathRef(clientPath))
+    }
+
+}
+
+// Documentation for mill.example.javascriptlib
+
+/** Usage
```

Comment:
This usage block is too verbose. Just showing/asserting the key lines like `Test Suites:...1 passed, 1 total...` is enough